releasenotes/notes/backendv1-d0d0642ed38fed3c.yaml

```diff
@@ -0,0 +1,7 @@
+---
+deprecations_providers:
+  - |
+    BackendV1 is deprecated and it will be removed not earlier than the next major release.
+    If providers want to explose a backend only as a hardware description, consider constructing a :class:`.Target`.
+    If the provider also provides access for execution, that can be done via the primitives interfaces.
+    Alternatebly, if the previous model is prefered, consider moving to :class:`.BackendV2` (see https://qisk.it/backendV1-to-V2).
```

Comment:
```suggestion
    The ``BackendV1`` class is deprecated and it will be removed not earlier than the next major release.
    There are several migration paths available depending on the main purpose of the backend object:
        - To expose backend information with no access to execution (just a hardware description), consider constructing a :class:`.Target` directly.
        - To provides access to execution capabilities, consider relying on the primitives interfaces instead.
        - Alternatively, to continue providing simultaneous :class:`.Target` (hardware information) and ``run`` (execution) capabilities, consider moving to :class:`.BackendV2` (see https://qisk.it/backendV1-to-V2).
```